.github/workflows/releaser.yml

```diff
@@ -6,11 +6,33 @@ jobs:
     runs-on: ubuntu-latest
     env:
       VERSION: ""
+      CREATETAG: "false"
+      BRANCH: ""
     steps:
       - uses: actions/checkout@v2
       - name: Determine version
         run: echo "VERSION=$(jq -r .version version.json)" >> $GITHUB_ENV
+      - name: Determine branch
+        run: echo "BRANCH=refs/heads/${{ github.event.repository.default_branch }}" >> $GITHUB_ENV
+      - name: Create a release, if we're on the default branch
+        run: echo "CREATETAG=true" >> $GITHUB_ENV
+        if: env.BRANCH == github.ref
+      - name: Determine if this is commit is a merged PR (if we're not on a non-default branch)
```

Comment:
```suggestion
      - name: Determine if this commit is a merged PR (if we're not on a default branch)
```

Reply:
`if we're not on a default branch` because `env.BRANCH != github.ref`, right?

Should we rename `env.BRNACH` to `env.DEFAULT_BRANCH` to make it clearer?

Reply:
Yes, looking at this now with a fresh mind `env.BRANCH` seems like exceptionally poor naming. Thanks for catching this!

Reply:
Fixed.